bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java

```diff
@@ -311,6 +312,15 @@ private void configureMetricsBridge(Builder settings) {
     MetricsApiTracerAdapterFactory metricsApiTracerAdapterFactory =
         new MetricsApiTracerAdapterFactory();
     settings.stubSettings().setTracerFactory(metricsApiTracerAdapterFactory);
+    // Set up channel interceptor to record channels.channel$id.rpc.latency metrics
+    if (settings.stubSettings().getTransportChannelProvider()
+        instanceof InstantiatingGrpcChannelProvider) {
+      InstantiatingGrpcChannelProvider.Builder channelProvider =
+          ((InstantiatingGrpcChannelProvider) settings.stubSettings().getTransportChannelProvider())
+              .toBuilder();
+      channelProvider.setInterceptorProvider(new MetricsChannelInterceptorInterceptorProvider());
```

Comment:
Note: Not sure about this part.. this won't include the interceptors that's already set in the setting, but looks like there isn't a way in InstantiatingGrpcChannelProvider to get interceptor provider from the builder https://github.com/googleapis/gax-java/blob/master/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java#L468-L471?